Source/cmServerProtocol.cxx

```diff
@@ -649,11 +650,53 @@ struct hash<LanguageData>
 
 } // namespace std
 
-static Json::Value DumpCTestInfo(std::pair<std::string, std::string> &testInfo)
+
+static Json::Value DumpBacktrace(const cmListFileBacktrace& backtrace)
```

Comment:
Did you change DumpBacktrace? It's weird it's showing up in the diff?

Reply:
No I just moved it. It was below a function I needed to use it in.